x/dfract/keeper/grpc_query.go

```diff
@@ -7,9 +7,10 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/cosmos/cosmos-sdk/types/query"
-	"github.com/lum-network/chain/x/dfract/types"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+
+	"github.com/lum-network/chain/x/dfract/types"
 )
 
 var _ types.QueryServer = Keeper{}
```

Comment:
Is there a reason why we don't unify the NewQueryServerImpl here as in beams and millions ?

```
func NewQueryServerImpl(keeper Keeper) types.QueryServer {
	return queryServer{Keeper: keeper}
}
```